src/main/resources/hazelcast-config-schema.json

```diff
@@ -1373,22 +1373,129 @@
             "$ref": "#/definitions/AsyncBackupCount"
           },
           "item-listeners": {
-          "type": "array",
-            "items":  {
+            "type": "array",
+            "items": {
               "$ref": "#/definitions/ItemListener"
             }
           },
           "split-brain-protection-ref": {
             "type": "string",
-            "description":  "Adds the Split Brain Protection for this data-structure. You should set this value as a \"split-brain-protection\"'s name."
+            "description": "Adds the Split Brain Protection for this data-structure. You should set this value as a \"split-brain-protection\"'s name."
           },
           "merge-policy": {
             "$ref": "#/definitions/MergePolicy"
           }
         }
       }
+    },
+    "NativeMemory": {
+      "allOf": [
+        {
+          "description": "This feature is available only in Hazelcast Enterprise HD. Configuration element's name is \"native-memory\".",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "default": false,
+              "description": "Set as true to enable the High-Density Memory Store usage. Its default value is false."
+            },
+            "allocator-type": {
+              "type": "string",
+              "enum": [
+                "STANDARD",
+                "POOLED"
+              ],
+              "default": "POOLED",
+              "description": "Type of the memory allocator. The default value is POOLED. Available values are as follows: \n STANDARD: Allocates/frees the memory using default OS memory manager, \n POOLED: Manages memory blocks in thread local pools"
```

Comment:
nit: if you write \n as \\n then IJ will show the line break when showing up the documentation.

Reply:
fixed